src/BotPlutusInterface/Files.hs

```diff
@@ -295,7 +295,7 @@ mkDummyPrivateKey (PubKey (LedgerBytes pubkey)) =
         Crypto.xprv $
           mconcat [dummyPrivKey, dummyPrivKeySuffix, pubkeyBS, dummyChainCode]
 
-serialiseScript :: Script -> PlutusScript PlutusScriptV1
+serialiseScript :: Script -> PlutusScript PlutusScriptV2
```

Comment:
Will V1 scripts keep working with this change?

Reply:
No it won't work with V1. Core things is difference in V1/V2 ScriptContext and validatorHash